File: 1-Iniciante/ex1010.py
#entrada
codigo, qtd, valor = str(input()).split()
codigo2, qtd2, valor2 = str(input()).split()
#processamento

codigo = int(codigo)
codigo2 = int(codigo2)
qtd = int(qtd)
qtd2 = int(qtd2)
valor = float(valor)
valor2 = float(valor2)

total = (valor * qtd) + (valor2 * qtd2)
#saída
print("VALOR A PAGAR: R$ {:.2f}".format(total))

File: 1-Iniciante/ex1010.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd

// processamento
let piece1 = input.split('\n')[0].split(' ').map(n => parseFloat(n));
let piece2 = input.split('\n')[1].split(' ').map(n => parseFloat(n));

let total = piece1[1] * piece1[2] + piece2[1] * piece2[2];

// saida
console.log('VALOR A PAGAR: R$ ' + total.toFixed(2));